server/services/participants.js

```diff
@@ -133,6 +134,20 @@ const updateParticipant = async (participantInfo) => {
     },
     { history: participantInfo.history, userUpdatedAt: new Date().toJSON() }
   );
+  if(changes.interested !== undefined){
```

Comment:
nitpick - the spacing here is inconsistent. I thought our linting should catch this - any idea if it didn't run / isn't looking for this?

Reply:
For whatever reason my linting doesn't run on commit and I forgot to run the formatter. Good catch!